uqcsbot/scripts/yt.py

```diff
@@ -0,0 +1,54 @@
+import os
+from uqcsbot import bot, Command
+import argparse
+
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+
+YOUTUBE_API_KEY = 'AIzaSyCtxVMs6So6x2WL5WkDV4rm01hzddCCGH4'
+YOUTUBE_API_SERVICE_NAME = 'youtube'
+YOUTUBE_API_VERSION = 'v3'
+YOUTUBE_VIDEO_URL = 'https://www.youtube.com/watch?v='
+
+
+@bot.on_command('yt')
+def handle_yt(command: Command):
+    '''
+    `!yt <QUERY>` - Returns the top search result based on the query string.
+    
+    handle_yt checks the usage of yt command and appropriately selects logic.
```

Comment:
Ideally, the helper doc should implicitly specify function functionality -- which in this case it looks like it does, so this additional docstring can probably be removed. (Additionally, it gets included in the helper doc `!help yt`, which makes it read strange)